apps/web/public/static/locales/en/common.json

```diff
@@ -19,10 +19,10 @@
   "need_to_reschedule_or_cancel": "Need to reschedule or cancel?",
   "cancellation_reason": "Reason for cancellation (optional)",
   "cancellation_reason_placeholder": "Why are you cancelling?",
-  "rejection_reason": "Reason for rejecting",
-  "rejection_reason_title": "Reject the booking request?",
-  "rejection_reason_description": "Are you sure you want to reject the booking? We'll let the person who tried to book know. You can provide a reason below.",
-  "rejection_confirmation": "Reject the booking",
+  "rejection_reason": "Reason for declination",
```

Comment:
Declination isn't really a word

Reply:
https://www.dict.cc/?s=declination its not? 👀

Reply:
<img width="1054" alt="Screenshot 2023-02-19 at 2 38 02 PM" src="https://user-images.githubusercontent.com/1780212/219939075-19f9b3be-8eda-4576-82bb-80bbdf10c1e0.png">
Seems like a word with completely different meaning 😅

Reply:
true!